### PR TITLE
typescript things

### PR DESCRIPTION
**Pull request by DalThePal**

### CODE REVIEW

File: RichText/index.tsx [hunk 1]
```diff
@@ -26,15 +23,22 @@ import {
...
-    | NonNullable<
-        PageProps<Queries.BlogArticleQuery>["data"]["contentfulBlogArticle"]
-      >["content"]
-    | NonNullable<
-        PageProps<Queries.CaseStudyQuery>["data"]["contentfulCaseStudy"]
-      >["content"]
+// TODO: YOU BETTER NOT BE USING THIS FOR ANY PROJECTS
+interface Props {
+  // eslint-disable-next-line
+  content: any
```
Comment (RJWadley) [inline RichText/index.tsx:41]: change this to unknown and it should be fine

File: RichText/index.tsx [hunk 2]
```diff
@@ -26,15 +23,22 @@ import {
...
-type Props = {
-  content:
-    | NonNullable<
-        PageProps<Queries.BlogArticleQuery>["data"]["contentfulBlogArticle"]
-      >["content"]
-    | NonNullable<
-        PageProps<Queries.CaseStudyQuery>["data"]["contentfulCaseStudy"]
-      >["content"]
+// TODO: YOU BETTER NOT BE USING THIS FOR ANY PROJECTS
+interface Props {
```
Comment (RJWadley) [inline RichText/index.tsx:39]: prefer more descriptive type names like RichTextProps

File: RichText/index.tsx [hunk 3]
```diff
@@ -26,15 +23,22 @@ import {
   H5,
   H6,
 } from "./modules/StyledComponents"
+import renderContent from "./renderContent"
+
+// type Props = {
```
Comment (RJWadley) [inline RichText/index.tsx:28]: you can remove this

File: RichText/index.tsx (hunk 1 above)
Comment (RJWadley) [reply RichText/index.tsx:41]: or maybe theres a general gatsby type we can take advantage of?

**Review by RJWadley — APPROVED** [review]

File: RichText/index.tsx (hunk 2 above)
Comment (DalThePal) [reply RichText/index.tsx:39]: well i prefer the opposite

**Pull request merged**